lib/winston-loggly.js

```diff
@@ -58,7 +58,7 @@ var Loggly = exports.Loggly = function (options) {
     bufferOptions: options.bufferOptions || {size: 500, retriesInMilliSeconds: 30 * 1000}
   });
 
-  this.timestamp = options.timestamp || false;
```

Comment:
@Shwetajain148 this means that timestamp is always true even if options contain it as false. The behavior we implement is by default it is true and if user specify it as false then we keep it false.